libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -741,17 +740,43 @@ void CompilerUtils::convertType(
 		else if (targetTypeCategory == Type::Category::FixedPoint)
 		{
 			solAssert(
-				stackTypeCategory == Type::Category::Integer || 
+				stackTypeCategory == Type::Category::Integer ||
 				stackTypeCategory == Type::Category::RationalNumber ||
 				stackTypeCategory == Type::Category::FixedPoint,
 				"Invalid conversion to FixedMxNType requested."
 			);
-			//shift all integer bits onto the left side of the fixed type
+
 			FixedPointType const& targetFixedPointType = dynamic_cast<FixedPointType const&>(_targetType);
-			if (auto typeOnStack = dynamic_cast<IntegerType const*>(&_typeOnStack))
-				if (targetFixedPointType.numBits() > typeOnStack->numBits())
-					cleanHigherOrderBits(*typeOnStack);
-			solUnimplemented("Not yet implemented - FixedPointType.");
+			if (stackTypeCategory == Type::Category::Integer)
+			{
+				IntegerType const& typeOnStack = dynamic_cast<IntegerType const&>(_typeOnStack);
+				cleanHigherOrderBits(typeOnStack);
+				m_context << u256(pow(bigint(10), targetFixedPointType.fractionalDigits()))
+						  << Instruction::MUL;
```

Comment:
Line: either place this on the same line or have a new `m_context << `here or move both onto a separate line from `m_contaxt`.